src/Charcoal/Property/ImageProperty.php

```diff
@@ -325,6 +346,10 @@ public function save($val)
             $val = $this->processEffects($val);
         }
 
+        if ($val) {
+            $this->imageCompression->compress($val);
+        }
+
```

Comment:
If we using effects to create intermediate images (what we expect to show on the web site), why compress the the selected (original) image?

If we want to compress the original image, we should be attaching the image compression service to the [`ElfinderConnectorAction`](https://github.com/locomotivemtl/charcoal-admin/blob/0.29.0.1/src/Charcoal/Admin/Action/ElfinderConnectorAction.php) in charcoal-admin instead of charcoal-property.